test/lint/lint-python.sh

```diff
@@ -33,7 +33,6 @@ enabled=(
     E401 # multiple imports on one line
     E402 # module level import not at top of file
     E502 # the backslash is redundant between brackets
-    E701 # multiple statements on one line (colon)
```

Comment:
Meh on this, I don't think a change of code style is necessary in this PR

Reply:
Agree, could lead to future style bikeshedding.

Reply:
Necessary or not, this is our current code style for C++ and seems quite reasonable...

Where is the basis for banning it? It's not like it's dangerous syntax.

Reply:
In C++, if code blocks are denoted by braces. Having a single line if statement indented without braces is dangerous because it's easy to misread a subsequent indented line as part of the if block. These things have caused bugs in production systems, which is why it style guide disallows it.

Python is a completely different language. If blocks are denoted by indentation. The same considerations don't apply.

Please remove this unnecessary style change.